functional-tests/http/.java-version

```diff
@@ -0,0 +1 @@
+1.8
```

Comment:
This file addition doesn't sound intentional

Reply:
Oops! Sorry, removed